### PR TITLE
Atomizes flamethrower and incendiary grenade crates

### PR DESCRIPTION
**Pull request by zimon9**
## About The Pull Request

This change splits the incendiary cargo crate, containing 3 flamethrowers and 3 incendiary grenades for 5000 credits, into two crates: 

| Flamethrower Crate | Incendiary Grenade Crate |
|--------------------|--------------------------|
| 1x flamethrower    | 1x incendiary grenade    |
| 1250 credits       | 750 credits              |

If added together, this brings the total cost, if one were to replicate the kit, to 6000 credits.

## Why It's Good For The Game

This crate was long overdue for atomization, just like how all of the weapon crates available right now have already been atomized.

## Changelog

:cl:
add: Added flamethrower crate
add: Added incendiary grenade crate
del: Removed incendiary supply crate
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs/sec_supply.dm [hunk 1]
```diff
@@ -90,19 +90,25 @@
...
+	cost = 1250
+	contains = list(
+		/obj/item/flamethrower/full
+	)
+	crate_name = "flamethrower crate"
+	crate_type = /obj/structure/closet/crate/secure/weapon
+	faction = FACTION_NGR
+
+/datum/supply_pack/sec_supply/incendiary_grenade
+	name = "incendiary Grenade Crate"
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/sec_supply.dm]:
```suggestion
	name = "Incendiary Grenade Crate"
```

File: code/modules/cargo/packs/sec_supply.dm [hunk 2]
```diff
@@ -90,19 +90,25 @@
...
-	name = "Incendiary Weapons Crate"
-	desc = "Burn, baby burn. Contains three incendiary grenades and three prebuilt flamethrowers."
-	cost = 5000
+/datum/supply_pack/sec_supply/flamethrower
+	name = "Flamethrower Crate"
+	desc = "Contains one flamethrower. Point the nozzle away from anything important."
+	cost = 1250
+	contains = list(
+		/obj/item/flamethrower/full
+	)
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/sec_supply.dm]:
```suggestion
	contains = list(/obj/item/flamethrower/full)
```

File: code/modules/cargo/packs/sec_supply.dm [hunk 3]
```diff
@@ -90,19 +90,25 @@
...
+	desc = "Contains one incendiary grenade. Better not let it go off in your hands."
+	cost = 750
 	contains = list(
-		/obj/item/flamethrower/full,
-		/obj/item/flamethrower/full,
-		/obj/item/flamethrower/full,
-		/obj/item/grenade/chem_grenade/incendiary,
-		/obj/item/grenade/chem_grenade/incendiary,
-		/obj/item/grenade/chem_grenade/incendiary,
+		/obj/item/grenade/chem_grenade/incendiary
```
Comment (FalloutFalcon) [inline code/modules/cargo/packs/sec_supply.dm]: same . review wont let me suggest it

**Review by FalloutFalcon — CHANGES_REQUESTED** [review]
Otherwise perfect!

File: code/modules/cargo/packs/sec_supply.dm (hunk 1 above)
Comment (zimon9) [reply code/modules/cargo/packs/sec_supply.dm]: Thanks for catching that! Let me add the changes in

**Comment by zimon9** [comment]
There we go. It should be cleaner now. Thanks for the suggestions.

**Review by FalloutFalcon — APPROVED** [review]

**Pull request merged**